examples/tutorial_examples/use_subject_space_rois_from_freesurfer.py

```diff
@@ -0,0 +1,135 @@
+#
+# Retrieves High angular resolution diffusion imaging (HARDI) dataset from
+# Stanford's Vista Lab
+#
+#   see https://purl.stanford.edu/ng782rw8378 for details on dataset.
+#
+# The data for the first subject and first session are downloaded locally
+# (by default into the users home directory) under:
+#
+#   ``.dipy/stanford_hardi/``
+#
+# Anatomical data (``anat``) and Diffusion-weighted imaging data (``dwi``) are
+# then extracted, formatted to be BIDS compliant, and placed in the AFQ
+# data directory (by default in the users home directory) under:
+#
+#   ``AFQ_data/stanford_hardi/``
+#
+# This data represents the required preprocessed diffusion data necessary for
+# intializing the GroupAFQ object (which we will do next)
+#
+# The clear_previous_afq is used to remove any previous runs of the afq object
+# stored in the AFQ_data/stanford_hardi/ BIDS directory. Set it to false if
+# you want to use the results of previous runs.
+
+afd.organize_stanford_data(clear_previous_afq="track")
+
+##########################################################################
+# Generate left thalamus ROI from freesurfer segmentation file
+# ------------------------------------------------------------
+# 1. Load the segmentation file that was generated by Freesurfer for
+#    the specific subject.
+# 2. Identify the left thalamus within the file, which has the label
+#    number 41
+# 3. Create a Nifti image representing the left thalamus ROI:
+#    - Assign a value of 1 to the pixels that Freesurfer
+#      has labeled as 41 (i.e., the left thalamus).
+#    - Assign a value of 0 to all other pixels.
+# This binary mask format is the expected input for pyAFQ when
+# dealing with subject space ROIs. If it's already in binary format,
+# there is no need to do this step.
+
+freesurfer_subject_folder = op.join(
+    afd.afq_home, "stanford_hardi",
+    "derivatives", "freesurfer",
+    "sub-01", "ses-01",
+    "anat")
+
+seg_file = nib.load(op.join(
+    freesurfer_subject_folder, "sub-01_ses-01_seg.nii.gz"))
+left_thal = seg_file.get_fdata() == 41
+nib.save(
+    nib.Nifti1Image(
+        left_thal.astype(np.float32),
+        seg_file.affine),
+    op.join(
+        freesurfer_subject_folder,
+        "sub-01_ses-01_desc-leftThal_mask.nii.gz"))
+
+##########################################################################
+# Set tractography parameters (optional)
+# ---------------------
+# We make this tracking_params which we will pass to the GroupAFQ object
+# which specifies that we want 10,000 seeds randomly distributed
+# only within the endpoint ROIs and not throughout the white matter.
+# This is controlled by passing
+# `"seed_mask": RoiImage()` in the `tracking_params` dict.
+#
+# We only do this to make this example faster and consume less space.
+
+tracking_params = dict(n_seeds=10000,
+                       random_seeds=True,
+                       rng_seed=42,
+                       seed_mask=RoiImage(use_endpoints=True))
```

Comment:
Do I understand correctly that tracking will only happen if it hasn't happened in other examples?

Reply:
Whats being kept is basically the mapping and already fit models.